classrooms/models.py

```
@@ -1,3 +1,83 @@
+        unique_together = ['classroom', 'teacher']
+
+    def __str__(self):
+        return f"{self.teacher.username} teaches {self.classroom.title}"
+
+
+class Enrollment(BaseModel):
+    STATUS_CHOICES = [
+        ('active', 'Active'),
+        ('pending', 'Pending'),
+        ('withdrawn', 'Withdrawn'),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)